### PR TITLE
ui/core/config.sandpit.json

### PR DESCRIPTION
**Pull request by 59023g**
This enables Keplr chain suggestion.

### CODE REVIEW

**Review by ryardley — DISMISSED** [review]

File: ui/core/src/config.sandpit.json [hunk 1]
```diff
@@ -2,8 +2,43 @@
...
   "sifApiUrl": "https://lcd-sandpit.sifchain.finance/",
   "sifWsUrl": "wss://rpc-sandpit.sifchain.finance/websocket",
+  "sifRpcUrl": "https://rpc-sandpit.sifchain.finance/",
   "sifChainId": "sandpit",
   "web3Provider": "metamask",
   "nativeAsset": "rowan",
-  "bridgebankContractAddress": "0xa5658e85bc7AC7F0388E60E58BEd5aFB41143330"
+  "bridgebankContractAddress": "0xa5658e85bc7AC7F0388E60E58BEd5aFB41143330",
+  "keplrChainConfig": {
+    "chainId": "sifchain",
```
Comment (utx0) [inline ui/core/src/config.sandpit.json:11]: the chainID for sandpit = `sandpit`

File: ui/core/src/config.sandpit.json [hunk 2]
```diff
@@ -2,8 +2,43 @@
...
+    "feeCurrencies": [
+      {
+        "coinDenom": "ROWAN",
+        "coinMinimalDenom": "rowan",
+        "coinDecimals": 18
+      }
+    ],
+    "coinType": 118,
+    "gasPriceStep": {
+      "low": 0.01,
```
Comment (utx0) [inline ui/core/src/config.sandpit.json:39]: With the next deployment we are seeing the min gas price to 0.5 rowan https://sifchain.slack.com/archives/C01ANGGQBQS/p1610580693017600?thread_ts=1610534084.011000&cid=C01ANGGQBQS

Comment (utx0) [reply ui/core/src/config.sandpit.json:39]: for the sifchain side that is.

Comment (utx0) [reply ui/core/src/config.sandpit.json:38]: Once we get sandpit upgraded we will need to do some testing around the gas fees.

File: ui/core/src/config.localnet.json [hunk 3]
```diff
@@ -8,8 +8,10 @@
   "nativeAsset": "rowan",
   "bridgebankContractAddress": "0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4",
   "keplrChainConfig": {
-    "chainId": "sifchain",
     "chainName": "Sifchain",
+    "chainId": "",
```
Comment (ryardley) [inline ui/core/src/config.localnet.json:12]: should this be "sifchain" or was that causing problems?

Comment (59023g) [reply ui/core/src/config.localnet.json:12]: I changed it so only defined in one place. See parseConfig.ts

Comment (59023g) [reply ui/core/src/config.localnet.json:12]: I changed it so `chainId` only defined in one place. See parseConfig.ts

File: ui/core/src/config.sandpit.json [hunk 4]
```diff
@@ -2,8 +2,45 @@
...
   "sifWsUrl": "wss://rpc-sandpit.sifchain.finance/websocket",
+  "sifRpcUrl": "https://rpc-sandpit.sifchain.finance/",
   "sifChainId": "sandpit",
   "web3Provider": "metamask",
   "nativeAsset": "rowan",
-  "bridgebankContractAddress": "0xa5658e85bc7AC7F0388E60E58BEd5aFB41143330"
+  "bridgebankContractAddress": "0xD67F01EdA3541BBf0D17A87BBe8B30E21A822B26",
+  "keplrChainConfig": {
+    "chainName": "Sifchain",
+    "chainId": "",
```
Comment (utx0) [inline ui/core/src/config.sandpit.json:12]: this should be `sandpit`

Comment (ryardley) [reply ui/core/src/config.sandpit.json:12]: It is set in the `parseConfig` function based on the `sifChainId` above. Could we remove it here @59023g ?

Comment (59023g) [reply ui/core/src/config.sandpit.json:12]: It's defined here, then object is updated in parseconfig.ts https://github.com/Sifchain/sifnode/pull/493/files/72ead783f2b5909bb9b91920da2bc2e468865e9a#diff-1f2cf9825b529857b03f1e86239bcc8fdd2000da15ae45a590a801cf6bb5bc6bL5

**Review by ryardley — APPROVED** [review]

File: ui/core/src/config.sandpit.json (hunk 4 above)
Comment (utx0) [reply ui/core/src/config.sandpit.json:12]: yeah ok.

**Pull request merged**